### DISCUSSION TITLE
Application doesnt respond properly anymore

### DISCUSSION THREAD

**Discussion by wowogiengen** [TumblThreeApp/TumblThree#432] [Q&A (Question & Answer)] [answered]
Hello,
maybe its just a beginner issue, but i observe that the application doesnt respond to ALT+F4 or the close icons/buttons, and i have to kill it via task manager. 

I already managed to add a whole lot of tumblr blogs (around 30 items), and they have been downloading also a huge amount of images, blog entries and so on. 

But how am i supposed to update / download new items of the blogs?
Because the download / pause / continue and stop button on the bottom seem to do nothing.
Is there a way to enable progress  visualisation or some other kind of debug log?
I can debug with visual studio, but i am not very good in WPF programming, so still very unsure whats going on and what not

What is needed that somebody can help me?

**Comment by thomas694** [comment, marked as answer]
Hello,
can you redescribe what does "anymore" mean? Since when or what action?

30 blogs isn't that much or too much so the app should hang.
For downloading a blog you need to add it to the queue, but you should already know that because you say you downloaded already a bunch of things. If blogs shall be updated you need to add them first to the queue again.
If you can press the start/stop buttons, I would say the queue is empty.
If you start the app and can't press the buttons, give the app a few seconds (or a minute) to properly load all the blogs from disk.

You can enable the `TumblThree.log` (folder %LOCALAPPDATA%\TumblThree\) in the settings.
Or when starting the download it should go through `CrawlerController.RunCrawlerTasksAsync()`.
